_episodes/02-vcs.md

```diff
@@ -7,94 +7,93 @@ questions:
-- "Using VCS does not only improves sharing and collaboration, but
-  also could be critical to assist and to guarantee reproducibility"
-- "VCS could be used directly or serve a foundation for domain-specific tools"
+- "Using VCS not only improves sharing and collaboration, but
+  also is integral to assisting and guaranteeing reproducibility"
+- "VCS can be used directly or can serve a foundation for domain-specific tools"
 ---
 
-> ## You can skip this lesson if you can answer these questions? --->
+> ## You can skip this lesson if you can answer these questions --->
 >
->  - How to keep all your scripts, configuration, notes, and data
+>  - How do you keep all your scripts, configuration, notes and data
 >    under version control systems and shareable with your collaborators?
->  - How to establish and use continuous integration systems to verify correctness
->    of reproduce results (where feasible)?
+>  - How do you establish and use continuous integration systems to verify the correctness
+>    of reproduced results (where feasible)?
 >  - What exactly have you done in your sample data analysis project X on a date Y?
 {: .challenge}
 
 
 ## What is a “version control system" ([VCS])?
 
-We all probably do some level of version control of our files,
+We all probably do some level of version control with our files,
 documents, and even data files, but without a version control **system** ([VCS])
 we do it in an ad-hoc manner:
 
 [![A Story Told in File Names by Jorge Cham, http://www.phdcomics.com/comics/archive_print.php?comicid=1323](../fig/borrowed/phd052810s.png)](http://www.phdcomics.com)
 
-So, in general, VCS systems help to **track versions** of digital artifacts
+So, in general, VCSs help **track versions** of digital artifacts
 such as code (scripts, source files), configuration files, images,
-documents, data -- original or generated (as outcome of the analysis).
-With proper annotation of changes, VCS becomes the lab notebook for
+documents, and data -- both original or generated (as outcome of the analysis).
+With proper annotation of changes, a VCS becomes the lab notebook for
 changing content in the digital world.  Since all versions are stored,
 VCS makes it possible to provide any previous version at any later
-point in time.  You can see how it could be important for reproducing
+point in time.  You can thus see how it can be important for reproducing
 previous results -- if your work's history is stored in a VCS, you just
 need to get a previous version of your materials and carry out the
 analysis using it.  You can also recover a file which you mistakenly
-removed since previous version would be contained within VCS, so no
-more "the cat ate my source code". For those features alone it is
-worthwhile placing any materials you produce and care about under
+removed since a previous version would be contained within your VCS, so no
+more excuses like “the cat ate my source code". For those features alone it is
```

Comment:
again rogue `“` -- just replace them all. This is a helper I use 
```shell
$> cat bin/git-sedi 
#!/bin/bash

git grep -l "$1" | xargs sed -i -e "s/$1/$2/g"
```
so I would 
1. commit all the changes
2. `git-sedi '“' '"'`
3. verify using `git diff --color-words` that the changes are correct
4. `git commit`